ESP8266-PIR-OLED-Arduino-OTA.ino

```diff
@@ -0,0 +1,339 @@
+//DHT, Motion and OLED sketch for "The Kube" sensor
+
+//by bkpsu, https://www.thingiverse.com/thing:2539897
+
+#define MQTT_SOCKET_TIMEOUT 120
+
+/************ HARDWARE CONFIG (CHANGE THESE FOR YOUR SENSOR SETUP) ******************/
+#define REMOTE //Uncomment to enable remote sensor functionality (Wifi & MQTT)
+//#define OLED_SPI //Uncomment if using SPI OLED screen (assuming I2C otherwise)
+#define CELSIUS //Uncomment if you want temperature displayed in Celsius
+//#define DEEP_SLEEP //Uncomment if you want sensor to sleep after every update (Does NOT work with MOTION_ON or LED_ON which require constant uptime)
+#define FLIP_SCREEN //Uncomment if mounting to wall with USB connector on top
+#define MOTION_ON //Uncomment if using motion sensor
+//#define OLED_MOTION //Uncomment if you want screen to turn on only if motion is detected
+//#define LED_ON //Uncomment if using as LED controller
+//#define PRESS_ON //Uncomment if using as Pressure monitor
+#define OTA_UPDATE //Uncomment if using OTA updates
+
+/************ WIFI, OTA and MQTT INFORMATION (CHANGE THESE FOR YOUR SETUP) ******************/
+#define wifi_ssid "wifi_ssid" //enter your WIFI SSID
+#define wifi_password "wifi_password" //enter your WIFI Password
+#define mqtt_server "mqtt_server" // Enter your MQTT server address or IP.
+#define mqtt_device "mqtt_device" //MQTT device
+#define mqtt_user "" //enter your MQTT username
+#define mqtt_password "" //enter your password
+#define OTApassword "123" // change this to whatever password you want to use when you upload OTA
+#define OTANAME "Esp-pir-oled"
```

Comment:
Let's use mqtt_device instead of creating another item that needs to be kept track of

Reply:
in the interest of this, how about we leave the port as the default? entirely up to you